apps/opik-backend/src/test/java/com/comet/opik/api/resources/v1/priv/ExperimentsResourceTest.java

```diff
@@ -1626,6 +1627,87 @@ void getById() {
         }
     }
 
+    @Nested
+    @TestInstance(TestInstance.Lifecycle.PER_CLASS)
+    class DeleteExperimentItems {
+
+        @Test
+        void deleteExperimentsById__whenExperimentExists__thenReturnNoContent() {
+            var experiment = podamFactory.manufacturePojo(Experiment.class);
+
+            createAndAssert(experiment, API_KEY, TEST_WORKSPACE);
+
+            deleteExperimentAndAssert(Set.of(experiment.id()), API_KEY, TEST_WORKSPACE);
+        }
+
+        private void deleteExperimentAndAssert(Set<UUID> ids, String apiKey, String workspaceName) {
+            try (var actualResponse = client.target(getExperimentsPath())
+                    .path("delete")
+                    .request()
+                    .header(HttpHeaders.AUTHORIZATION, apiKey)
+                    .header(WORKSPACE_HEADER, workspaceName)
+                    .post(Entity.json(new ExperimentsDelete(ids)))) {
+
+                assertThat(actualResponse.getStatusInfo().getStatusCode()).isEqualTo(204);
+            }
+
+            ids.parallelStream().forEach(id -> getExperimentAndAssertNotFound(id, apiKey, workspaceName));
```

Comment:
Minor: you could save multiple calls by finding the experiments e.g: by the same dataset_id.

Reply:
I thought about it, but then we would be checking the link with the `dataset_id` instead if the id exists

Reply:
You can assert afterwards on the ID as well. Anyway, not a big deal.